packages/react/src/components/global-banner/global-banner.tsx

```diff
@@ -204,24 +125,68 @@ const ButtonContainer = styled.div<{ isMobile: boolean }>`
     }
 `;
 
-const getIconName = (bannerType: GlobalBannerType): IconName => {
-    switch (bannerType) {
-        case 'alert':
-            return 'alertOctagon';
-        case 'warning':
-        case 'default':
-            return 'alertTriangle';
-        case 'info':
-            return 'info';
-    }
-};
+type GlobalBannerButtonState = 'default' | 'hover';
+
+interface ActionButtonProps {
+    bannerType: GlobalBannerType;
+}
 
 export interface ActionButton {
     label: string;
 
     onClick(event: MouseEvent<HTMLButtonElement>): void;
 }
 
+function getActionButtonColors(
+    { bannerType, theme, state }: StyledProps<{
+        bannerType: GlobalBannerType;
+        state: GlobalBannerButtonState
+    }>,
+): FlattenSimpleInterpolation {
```

Comment:
Je pense que je sortirais le state de l'objet pour utiliser les props `ActionButtonProps` directement
```suggestion
function getActionButtonColors(
    { bannerType, theme }: StyledProps<ActionButtonProps>,
    state: GlobalBannerButtonState,
): FlattenSimpleInterpolation {
```